app/src/main/java/edu/uwstout/p2pchat/ExpandableSettingsListViewAdapter.java

```diff
@@ -311,20 +319,57 @@ public void onClick(View view)
                 });
                 break;
             case 3:
-                //TODO Help page goes here.
```

Comment:
The fact that you are using a switch case is very problematic. If I wanted to switch the order of the settings, this entire switch case would have to be re-written. The code is not built in such a way that you can change it easily later.